pkg/stanza/fileconsumer/file.go

```diff
@@ -178,7 +196,9 @@ func (m *Manager) consume(ctx context.Context, paths []string) {
 		wg.Add(1)
 		go func(r *reader.Reader) {
 			defer wg.Done()
+			m.readingFiles.Add(ctx, 1)
```

Comment:
There's another place in `file_other.go` where we sometimes also call `ReadToEnd`

Reply:
Thanks! Shall we expand it for [windows](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d38caf1fa8bbfdb4302d06bc73ddeb11b39619ca/pkg/stanza/operator/input/windows/operator.go#L138) as well?

Reply:
On windows, we don't have this additional read opportunity because we close files as soon as we finish reading them.